.github/workflows/release.yml

```diff
@@ -35,13 +35,17 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        golang_cross: [ 1.16, 1.16-el7 ]
+        golang_cross: [ 1.16, 1.16-el7, 1.19-bullseye ]
```

Comment:
Shouldn't this just be 1.19-bullseye? AIUI, RHEL7 is not supported anymore so we don't needseparate builds as we did before.